README.md

```diff
@@ -60,7 +60,7 @@ A utility script (./tpccbenchmark) is provided for running the benchmark. The op
    --runscript &lt;arg&gt;         Run an SQL script
 -s,--sample &lt;arg&gt;            Sampling window
 -v,--verbose                 Display Messages
-```
+</pre>
```

Comment:
@kerneljake is this change necessary? what does it do functionally? my understanding is that the previous was valid markdown as well?

Reply:
The previous markdown does not render the usage message as proper HTML in a browser, plus the text columns are not aligned.  Backticks are syntactically correct markdown, but they yield literal &lt; when viewed in a browser, whereas <pre> renders angled brackets correctly.

Reply:
Hah! github markdown clobbered my previous comment.  I was trying to illustrate literal "ampersand ell tee semicolon" versus angled bracket.